server/athenian/api/controllers/jira_controller/__init__.py

```diff
@@ -458,6 +467,13 @@ async def _epic_flow(
                 epic.life_time = now - epic.created
                 if epic.work_began is not None:
                     epic.lead_time = now - epic.work_began
+
+            epic.acknowledge_time = resolve_acknowledge_time(
+                np.array([epic.created], dtype=np.datetime64),
+                np.array([epic.work_began], dtype=np.datetime64),
+                now,
+            )[0]
```

Comment:
Acc. 186 has 10k epics in 1 month and this will work too slow. We should try to vectorize over the df columns.